.github/workflows/classicpress-release.yml

```diff
@@ -0,0 +1,41 @@
+name: Check for ClassicPress release
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  release-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check release repository
+        run: |
+          echo "Collect and process json"
+          RELEASE_DATA=$(wget -cq "https://api.github.com/repos/ClassyBot/ClassicPress-nightly/releases?per_page=2" -O - | jq --compact-output 'limit( 1; .[] ) | .html_url' | sed 's/\"//g' | sed 's/\"//g')
+          CURRENT_VERSION=$(echo "${RELEASE_DATA}" | grep -Eo -m1 '[[:digit:]]\.[[:digit:]]\.[[:digit:]]')
+          RELEASE_DATE=$(echo "${RELEASE_DATA: -8}")
+          echo "current_version=${CURRENT_VERSION}" >> $GITHUB_ENV
+          echo "release_date=${RELEASE_DATE}" >> $GITHUB_ENV
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Compare to Repository
+        run: |
+          API_VERSION=$(grep -Eo -m1 '[[:digit:]]\.[[:digit:]]\.[[:digit:]]' ./v1/migration/index.php)
+          API_DATE=$(grep -Eo -m1 '[[:digit:]]{8}' ./v1/migration/index.php)
+          if [ ${{ env.current_version }} != ${API_VERSION} ]; then
+            sed "s|${API_VERSION}|${{ env.current_version }}|" ./v1/migration/index.php
+            sed "s|${API_DATE}|${{ env.release_date }}|" ./v1/migration/index.php
+          fi
+      - name: Create Pull Request
```

Comment:
What happens when there are no changes? Is it documented that this just silently does nothing, or does it fail somehow? Should we skip this step entirely, instead?

Reply:
According to the Workflow action documentation, if there are no changes it fails silently:
https://github.com/peter-evans/create-pull-request#action-behaviour

Reply:
Ok, since this is documented behavior, should be fine to leave as-is.